LICENSE

```diff
@@ -349,6 +350,19 @@
          Copyright 2012 Continuum Analytics, Inc.
 
 
+    =======================================================================================
+    Creative Commons Attribution 4.0 International (CC BY 4.0)
```

Comment:
Note that there are 6 different licenses as part of CC BY 4.0 (https://creativecommons.org/licenses/). It is important to know which one because some of them will prevent commercial usage, prevent derivative works, or require others to use the same license.

Reply:
provided link to exact license statement at the place that downloads it.